common-tools/clas-tracking/src/main/java/org/jlab/clas/tracking/kalmanfilter/AStateVecs.java

```diff
@@ -245,6 +253,81 @@ public class StateVec {
         public double energyLoss;
         public double dx;
         public double path;
+        
+        /////////////////////// extra variables for forward tracking ///////////////////////
+        // tx = px/pz
+        public double ty; //=py/pz
+        public double Q; //q/p
+        public double B;
+        public double deltaPath;   
+        public Matrix CM = new Matrix();;
```

Comment:
add a comment to remind to unify covMat and CM

Reply:
Comment is added.